week-01/assessment/capsuleHotel/src/Hotel.java

```diff
@@ -0,0 +1,142 @@
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class Hotel {
+
+    public static void main(String[] args) {
+        System.out.println("Welcome to the Capsules Hotel!");
+
+        Scanner console = new Scanner(System.in);
+
+        System.out.print("What is the number of Capsules? ");
+        int capsulesCount = 0;
+        capsulesCount = Integer.parseInt(console.nextLine());
+        System.out.println(capsulesCount);
+        String[] guestList = new String[capsulesCount];
+
+        System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
```

Comment:
I might add these as seperate print statements for readability 

```suggestion
        System.out.println("1. Check In");
        System.out.println("2. Check Out");
        System.out.println("3. View Guests");
        System.out.println("4. Exit");
        System.out.print("Choose on option [1-4]: ");
```